cypress/support/api_commands.js

```diff
@@ -233,3 +238,62 @@ Cypress.Commands.add('apiEnableOpenServer', (enable = true) => {
         body: config,
     });
 });
+
+/**
+ * Creates a new user via the API, adds them to 3 teams, and sets preference to bypass tutorial.
+ * Then logs in as the user
+ @param {Object} user - Object of user email, username, and password that you can optionally set. Otherwise use default values
+ @returns {Object} Returns object containing email, username, id and password if you need it further in the test
+ */
+Cypress.Commands.add('loginAsNewUser', ({email = `newE2ETestUser${Date.now()}@mattermost.com`, username = `NewE2ETestUser${Date.now()}`, password = 'password123'} = {}) => {
+    // # Login as sysadmin to make admin requests
+    cy.apiLogin('sysadmin');
+
+    // # Create a new user
+    return cy.request({method: 'POST', url: '/api/v4/users', body: {email, username, password}}).then((userResponse) => {
+        // Safety assertions to make sure we have a valid response
+        expect(userResponse).to.have.property('body').to.have.property('id');
+
+        const userId = userResponse.body.id;
+
+        // Get teams, select the first three, and add new user to that team
+        cy.request('GET', '/api/v4/teams').then((teamsResponse) => {
+            // Verify we have at least 2 teams in the response to add the user to
+            expect(teamsResponse).to.have.property('body').to.have.length.greaterThan(1);
+
+            const [team1, team2] = teamsResponse.body;
```

Comment:
Probably not an issue if we don't care what team will the user be member of as these `teamsResponse` will return teams if different order (sometimes the first two is the default teams, and sometimes not)

Reply:
I think in its current state it's fine. I imagine we'll end up having some scenarios where we want the new user to be a member of a particular team. When we need that, it can be another arg or property to support to add the user to those teams.